Instruments/config/2017/Skimmer.cfg

```diff
@@ -78,78 +64,84 @@ file: DYJetsToLL_M-50_HT-600to800.root
 file: DYJetsToLL_M-50_HT-800to1200.root
 file: DYJetsToLL_M-50_HT-1200to2500.root
 file: DYJetsToLL_M-50_HT-2500toInf.root
+file: DY4JetsToLL_M-50.root
+#file: DYJetsToLL_M-10to50.root DYJetsToLL_M-10to50_ext1.root DYJetsToLL_M-10to50_new_pmx.root
```

Comment:
- there should be a separate section for DY M-10to50
- check that `_new_pmx` samples is ok for us